csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependabotProxy.cs

```diff
@@ -1,14 +1,30 @@
 using System;
-using System.Diagnostics;
+using System.Collections.Generic;
 using System.IO;
 using System.Security.Cryptography.X509Certificates;
 using Semmle.Util;
 using Semmle.Util.Logging;
+using Newtonsoft.Json;
 
 namespace Semmle.Extraction.CSharp.DependencyFetching
 {
     public class DependabotProxy : IDisposable
     {
+        /// <summary>
+        /// Represents configurations for package registries.
+        /// </summary>
+        public struct RegistryConfig
```

Comment:
Is there any particular reason/benefit to declare this as a `struct`?

Reply:
No particular reason. I thought I was mirroring a similar setup elsewhere in the codebase, but happy to change it.